indico/modules/events/registration/templates/management/print_badges.html

```diff
@@ -42,4 +43,56 @@
             <input type="hidden" name="registration_id" value="{{ registration.id }}">
         {% endfor %}
     {% endcall %}
+
+    <script>
+        (function() {
+            'use strict';
+
+            var templates = {{ templates | tojson }};
+            var $template = $('#template');
```

Comment:
I think it's time for anything starting with this line to go into a JS file so all you need to do here is something like `setupBadgePrinting({{ templates | tojson }})` (if you can imagine adding more arguments to the function I'd go for the options dict like in other places)